api/client/info.go

```diff
@@ -33,7 +33,7 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 
 			// print a warning if devicemapper is using a loopback file
 			if pair[0] == "Data loop file" {
-				fmt.Fprintf(cli.err, " WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.")
+				fmt.Fprintf(cli.err, " WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.\n")
```

Comment:
thought about that but other warnings use `Fprintf` as well https://github.com/docker/docker/blob/master/api/client/info.go#L109

Should I change all the others as well?